lorawan-device/src/test_util.rs

```diff
@@ -1,11 +1,11 @@
 use super::*;
 use lorawan::maccommands::{ChannelMask, SerializableMacCommand};
-use lorawan::parser::DataHeader;
+use lorawan::parser::{self, DataHeader};
 use lorawan::{
     default_crypto::DefaultFactory,
     maccommandcreator::LinkADRReqCreator,
     maccommands::{LinkADRReqPayload, MacCommand},
-    parser::{parse, DataPayload, FCtrl, JoinAcceptPayload, PhyPayload},
+    parser::{parse, DataPayload, JoinAcceptPayload, PhyPayload},
```

Comment:
`FCtrl` is needed..

Try running tests with `-F async,external-lora-phy`

Reply:
I put it behind the async feature now